hs_core/hydroshare/utils.py

```diff
@@ -428,58 +416,38 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id, set_to_private=False):
     avu_list = ['bag_modified', 'metadata_dirty', 'isPublic', 'resourceType']
     src_res = get_resource_by_shortkey(src_res_id)
     tgt_res = get_resource_by_shortkey(dest_res_id)
+
+    # This makes the assumption that the destination is in the same exact zone.
+    # Also, bags and similar attached files are not copied.
```

Comment:
@alvacouch Yes, copying action intends to copy a resource to the same zone, which is a valid assumption I think. And yes, bags are intentionally not copied over since it is only generated on-demand when users want to download the bag.